com.sap.cloud.lm.sl.cf.process/src/main/resources/com/sap/cloud/lm/sl/cf/process/xs2-deploy.bpmn

```diff
@@ -156,6 +144,9 @@
     <callActivity id="deleteServicesCallActivity" name="Delete Services Sub Process" activiti:async="true" calledElement="deleteServicesSubProcess" activiti:inheritVariables="true"></callActivity>
     <sequenceFlow id="deleteDiscontinuedServicesFlow" sourceRef="shouldDeleteDiscontinuedServicesGateway" targetRef="deleteServicesCallActivity"></sequenceFlow>
     <sequenceFlow id="flow121" sourceRef="deleteServicesCallActivity" targetRef="createOrUpdateServiceBrokersTask"></sequenceFlow>
+    <callActivity id="createOrUpdateServicesCallActivity" name="Create or Update Services" activiti:async="true" calledElement="createOrUpdateServicesSubProcess" activiti:inheritVariables="true">
```

Comment:
add completeAsync="true" in order to the CallActivity to work :)

Reply:
Done.